### ISSUE TITLE
OmniAuth vulnerability found in Gemfile.lock

### ISSUE THREAD

**Issue by alexventuraio** [omniauth/omniauth-github#81]
Recently, I found that there is a vulnerability issue related to `omniauth` gem that is affecting `omniauth-github` gem too. I was wondering if there's any proposal to solve this issue?

https://nvd.nist.gov/vuln/detail/CVE-2015-9284

```
CVE-2015-9284 More information

high severity
Vulnerable versions: <= 1.9.0
Patched version: No fix
The request phase of the OmniAuth Ruby gem is vulnerable to Cross-Site Request Forgery when used as part of the Ruby on Rails framework, allowing accounts to be connected without user intent, user interaction, or feedback to the user. This permits a secondary account to be able to sign into the web application as the primary account.
```

<img width="1022" alt="Screen Shot 2019-10-03 at 09 59 13" src="https://user-images.githubusercontent.com/13169164/66138802-d7e57280-e5c4-11e9-81d7-8cf7b793494e.png">

**Comment by dentarg** [comment]
Perhaps you have already seen it, but the vulnerability has been discussed at length at https://github.com/omniauth/omniauth/pull/809 and there doesn't seem to be a clear way of fixing it in `omniauth`. Have a look at https://github.com/omniauth/omniauth/wiki/Resolving-CVE-2015-9284 if you haven't already.

With that complexity in mind and seeing that this gem, `omniauth-github`, [is looking for a new home since 2017](https://github.com/omniauth/omniauth-github/commit/2e77639f32c35598e398c0c591ffab37691cdee7), I wouldn't get my hopes up of anything happening here related to that issue.

**Comment by alexventuraio** [comment]
@dentarg can you take a look at [my problem here](https://github.com/cookpad/omniauth-rails_csrf_protection/issues/7), please?
Maybe you have any suggestions for me to fix my issue.
Thanks in advance!

**Comment by alexventuraio** [comment]
@BobbyMcWho one question, is this bug already fixed in the latest `v1.4.0` release?

**Comment by BobbyMcWho** [comment]
This is a separate issue with omniauth itself. Unfortunately it is a rather large undertaking, so there is not currently a plan in place to fix it.
